### PR TITLE
Update nissan_leaf.markdown

### PR DESCRIPTION
**Pull request by arallsopp**
## Proposed change
Clarified update interval as people were confusing this with polling frequencies and thinking a high value would refresh more frequently.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nissan_leaf.markdown [hunk 1]
```diff
@@ -113,6 +113,7 @@ You can also use the `nissan_leaf.update` service to request an on-demand update
 
 ## Hints
 
+* The `update_interval` is the time **between** updates. A high update interval will refresh rarely, and a low one refreshes more frequently.
```
Comment (ludeeus) [inline source/_integrations/nissan_leaf.markdown]: This is already described on line 48.

Comment (arallsopp) [reply source/_integrations/nissan_leaf.markdown]: I know. That's how I was able to guide the two people that were asking on social whether a low update interval would exascerbate or reduce the 12V load. If there's ambiguity, I'd seek to reduce it, but I appreciate a sample size of two probably isn't that significant.  Will defer to your preference  😃

Comment (ludeeus) [reply source/_integrations/nissan_leaf.markdown]: We should not add another line.
If line 48 needs adjusting, please adjust that instead.

**Comment by homeassistant** [comment]
Hi @arallsopp, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19453).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by homeassistant** [comment]
Hi @arallsopp, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19453).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/nissan_leaf.markdown (hunk 1 above)
Comment (arallsopp) [reply source/_integrations/nissan_leaf.markdown]: Believe that's now done.

**Review by frenck — APPROVED** [review]
Thanks, @arallsopp 👍

**Pull request merged**